gitlab/resource_gitlab_project_mirror.go

```diff
@@ -173,5 +173,10 @@ func resourceGitlabProjectMirrorSetToState(d *schema.ResourceData, projectMirror
 	d.Set("keep_divergent_refs", projectMirror.KeepDivergentRefs)
 	d.Set("only_protected_branches", projectMirror.OnlyProtectedBranches)
 	d.Set("project", projectID)
-	d.Set("url", projectMirror.URL)
+
+	if strings.Contains(projectMirror.URL, "*****") {
```

Comment:
that is not the best approach, you should parse the url and remove password in diff suppress func

Reply:
That sounds cleaner, but do you mean that we should save the redacted value in state? Wouldn't that make it impossible for the user to update the user/pass for their push mirror, since the diff would not be recognized?